pandas/core/arrays/base.py

```diff
@@ -280,6 +283,38 @@ def _from_sequence(cls, scalars, *, dtype: Dtype | None = None, copy: bool = Fal
         """
         raise AbstractMethodError(cls)
 
+    @classmethod
+    def _from_scalars(cls, scalars, *, dtype: DtypeObj) -> Self:
```

Comment:
Thinking aloud. Once we allow EA authors to specify their "is_scalar" passing equivalent, should that be incorporated here somehow?

Reply:
Yes.  Assuming at some point we add to the EADtype something like a `_is_recognized_scalar` method, the default implementation of that method might look like (assuming away NAs for the moment)

```
def _is_recognized_scalar(self, scalar) -> bool:
    cls = self.construct_array_class()
    try:
        cls._from_scalars([scalar], dtype=self)
    except TypeError:
        return False
    return True
```

We could alternately implement a default from_scalars in terms of _is_recognized_scalar.  _find_compatible_dtype from #53106 could also be the primitive from which the other two are constructed (though i dont think it can be constructed from either).